packages/devtools_app/lib/src/ui/analytics.dart

```diff
@@ -311,50 +325,49 @@ Future<bool> get isSurveyActionTaken async {
 // setSurveyActionTaken should only be called with the value of true, so
 // we can remove the extra complexity.
 void setSurveyActionTaken() async {
-  final resp = await HttpRequest.request(
-    // Format of request is e.g. api/setSurveyActionTaken?surveyActionTaken=true
-    '${server.apiSetSurveyActionTaken}'
-    '?${server.surveyActionTakenPropertyName}=true',
-    method: 'POST',
-  );
-  if (resp.status == HttpStatus.ok) {
-    assert(json.decode(resp.responseText) == true);
-  } else {
-    _logWarning(resp, server.apiSetSurveyActionTaken, resp.responseText);
+  if (isDevToolsServerAvailable) {
+    final resp = await _request(
+      '${server.apiSetSurveyActionTaken}'
+      '?${server.surveyActionTakenPropertyName}=true',
+    );
+    if (resp.status == HttpStatus.ok) {
+      assert(json.decode(resp.responseText) == true);
+    } else {
+      _logWarning(resp, server.apiSetSurveyActionTaken, resp.responseText);
+    }
   }
 }
 
 /// Request DevTools property value 'surveyShownCount' stored in the file
 /// '~\.devtools'.
 Future<int> get surveyShownCount async {
-  int surveyShownCount;
-
-  final resp = await HttpRequest.request(
-    '${server.apiGetSurveyShownCount}',
-    method: 'POST',
-  );
-  if (resp.status == HttpStatus.ok) {
-    surveyShownCount = json.decode(resp.responseText);
-  } else {
-    _logWarning(resp, server.apiGetSurveyShownCount);
+  int surveyShownCount = 0;
+
+  if (isDevToolsServerAvailable) {
+    final resp = await _request(server.apiGetSurveyShownCount);
+    if (resp.status == HttpStatus.ok) {
+      surveyShownCount = json.decode(resp.responseText);
+    } else {
+      _logWarning(resp, server.apiGetSurveyShownCount);
+    }
   }
+
   return surveyShownCount;
 }
 
 /// Increment DevTools property value 'surveyShownCount' stored in the file
 /// '~\.devtools'.
 Future<int> get incrementSurveyShownCount async {
   // Any failure will still return 1.
-  int surveyShownCount = 1;
-
-  final resp = await HttpRequest.request(
-    server.apiIncrementSurveyShownCount,
-    method: 'POST',
-  );
-  if (resp.status == HttpStatus.ok) {
-    surveyShownCount = json.decode(resp.responseText);
-  } else {
-    _logWarning(resp, server.apiIncrementSurveyShownCount);
+  int surveyShownCount = 0;
```

Comment:
The comment above doesn't match the value of 0.

Reply:
Thanks.